app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,262 @@
+---
+id: first_app.intellij-idea
+title: Building a Java application in IntelliJ IDEA
+slug: learn/intellij-idea
+type: tutorial
+category: start
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+more_learning:
+  youtube:
+    - H_XxH66lm3U
+    - 70_B2DyM8mU
+toc:
+- Overview
+- Installing IntelliJ IDEA
+- Creating a new project
+- Writing and editing code
+- Running your application
+- Testing
+- Debugging
+- Refactoring code
+- Documenting code
+- Searching and navigating
+- Summary
+description: "Learn how to code, run, test, debug and document a Java application faster in IntelliJ IDEA."
+last_update: 2024-04-04
+author: ["MaritvanDijk"]
+---
+
+## Overview
```

Comment:
The navigation doesn't work when running it locally because the table of contents doesn't know where the sections are. In the other articles, this is done by something like `<a id="overview">&nbsp;</a>` before the heading and then changing the `toc` to something like `- Overview {overview}`.

This also fixes the issue of the `This page was contributed by Marit van Dijk under the UPL` text being stuck to the heading without any space in between.

Reply:
Thanks, fixed